pkg/logger/logger.go

```diff
@@ -94,9 +99,10 @@ func NewLogger(
 	logReconciliation bool,
 	checkType CheckType,
 	network *types.NetworkIdentifier,
+	logRequestUUID string,
 	fields ...zap.Field,
 ) (*Logger, error) {
-	zapLogger, err := buildZapLogger(checkType, network, fields...)
+	zapLogger, err := buildZapLogger(checkType, network, logRequestUUID, fields...)
```

Comment:
@shrimalmadhur,  this part will make requestUUID to be a field in datadog?

Reply:
At least this should make the field as structured - we need to make sure the logs are emitted to datdog agent and not picked up via stdout

Reply:
@jingweicb we need to change this to Production logger - https://github.com/coinbase/rosetta-cli/blob/master/pkg/logger/logger.go#L118